trade-finance-manager-api/src/utils/date.ts

```diff
@@ -0,0 +1,56 @@
+import { format, isValid, parseISO } from 'date-fns';
+
+/**
+ * @param date 
+ * @returns Date as Unix timestamp representing the number of milliseconds between this date and 1st
+ * January 1970 (UTC), stored as a string.
+ */
+export const getDateAsEpochMillisecondString = (date: Date) => date.valueOf().toString();
+
+/**
+ * @returns Current date as Unix timestamp representing the number of milliseconds between now and 1st
+ * January 1970 (UTC), stored as a string.
+ */
+export const getNowAsEpochMillisecondString = () => getDateAsEpochMillisecondString(new Date());
+
+/**
+ * @param year as 2 or 4 digit number
+ * @returns year in `yyyy` formatting (e.g. 24 becomes 2024)
+ */
+export const formatYear = (year: string | number) => (Number(year) < 1000 ? (2000 + parseInt(String(year), 10)).toString() : year && year.toString());
+
+/**
+ * @param dateStr an ISO-8601 (or other) date string
+ * @returns date formatted as `yyyy-MM-dd` or 'Invalid date' if not a valid format
+ */
+export const formatDate = (dateStr: string) => {
+  const date = new Date(dateStr);
+
+  return isValid(date) ? format(date, 'yyyy-MM-dd') : 'Invalid date';
+};
+
```

Comment:
Genuinely no idea how to replicate this with date-fns.
The unit tests want the input `'20210419'` to be converted to `'2021-04-19'`
Haven't investigated how this is used across the code yet but probably should just look to remove this function entirely